clients/rospy/src/rospy/core.py

```diff
@@ -123,7 +124,7 @@ def parse_rosrpc_uri(uri):
     return dest_addr, dest_port
 
 #########################################################
-        

```

Comment:
Upstream will reject whitespace changes.

Reply:
these were automated by atom. i didn't deliberately make any of them. I'm not sure how to correct this